src/parser/sparqlParser/generated/SparqlAutomatic.g4

```diff
@@ -113,8 +113,11 @@ orderCondition
     | ( constraint | var )
     ;
 
-//This is a departure from the Sparql Specs.
-//We're adding an additional textLimitClause but usage is similar to the specs
+// This is an extension to the Sparql Specs.
+// We're adding an additional modifier to the limitOffsetClauses. This
+// modifier `TEXTLIMIT` is for QLever's full-text index. The order of the
```

Comment:
Sparql -> SPARQL

replace "is for" by "is used in" (... QLever's full-test...)

At the end of the comment the final dot `.` is missing.

(We are working on improving QLever's documentation and comments:)